src/tools/createAwsElasticSearchIndexesCli.js

```diff
@@ -0,0 +1,7 @@
+import createAwsElasticSearchIndexes from './createAwsElasticSearchIndexes';
```

Comment:
If we are going to make a CLI, shouldn't we add something to package.json to run it, in keeping with past practice?

Reply:
Also, I understand that we haven't done it before, but it would be nice to prompt for user confirmation/warn that this script will delete all data in the indexes in the interface since "createAwsIndexes" doesn't necessarily sound like it would do that.

Reply:
Oops... updated this.

Reply:
Should we prompt in the CLI tool that this will delete all data in the indexes before attempting to recreate them? In case we both leave the project and a new developer tries to run this.

Reply:
Also, trying to run this got: 

![Screen Shot 2022-12-16 at 9 43 00 AM](https://user-images.githubusercontent.com/3288586/208122929-16945bfd-0417-4603-9f72-53a2c61763b9.png)

Presumably I need to add info to my .env...

Can you add documentation either to the main readme or a new readme to the src/lib/awsElasticSearch folder detailing what needs to be done to set up elasticsearch locally?